### ISSUE TITLE
Add specific product card state for NEEDS PURCHASE

### ISSUE THREAD

**Issue by retrofox** [Automattic/jetpack#22941]
Currently, it is behaving in the same way of Plugin absent. We could have a different layout/messaging for it

**Comment by retrofox** [comment]
@keoshi maybe you can help us to implement it?

**Comment by keoshi** [comment]
What's the flow like? Don't we show the interstitial when clicking “Add Backup”?

**Comment by retrofox** [comment]
> What's the flow like? Don't we show the interstitial when clicking “Add Backup”?

yes, it shows the interstitial, but I think what @leogermani thinks is adding a different status in the card, in the Jetpack dashboard. Am I right, Leo?

**Comment by retrofox** [comment]
While I was re-organizing the code and dealing with some janitorial work got tempted to try something.a simple implementation. Maybe it can be a trigger to iterate with the idea.

![image](https://user-images.githubusercontent.com/77539/156553622-d0a06931-2082-445c-a52c-a85cbb7dac08.png)

![image](https://user-images.githubusercontent.com/77539/156553578-6e24233f-a94b-4189-bf38-2a4e2b33d130.png)

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/77539/156553418-97add00f-ed04-45b9-9351-dcd9b626d98a.png">
